src/common/marketUtils.es6

```diff
@@ -41,8 +61,16 @@ const getMarkets = (active_symbols) => {
     }, {});
 };
 
+function sortSubmarkets([...submarkets]) {
+    return submarkets.sort((a, b) => {
+        if (a < b) return -1;
```

Comment:
Why is the above sort sorting by display_name and not this one?

Reply:
`getMarketChild` is sorting the submarket as a whole object, used in header dropdown. however, `sortSubmarket` is sorting submarket as an array, used in asset index and trading times dropdown.

Reply:
Ah I see 👍